.github/actions/check-current-workflow-is-latest/action.yml

```diff
@@ -1,5 +1,7 @@
-name: Check current workflow is the latest workflow
-description: Checks whether the current workflow is the most recent workflow.
+name: Check Current Workflow Is The Latest Workflow
+description: >
+  Checks whether the current workflow is the most recent workflow for the pull request.
+  Intentionally not documented in Actions.md since it's only meant for internal use.
```

Comment:
```suggestion
  Checks whether the current workflow is the most recent workflow for the pull request. Intentionally not documented in
  Actions.md since it's only meant for internal use.
```